apps/api-extractor/src/api/model/ApiPropertySignature.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { ApiItemKind } from './ApiItem';
+import { ApiDeclarationMixin, IApiDeclarationMixinOptions } from '../mixins/ApiDeclarationMixin';
+import { ApiReleaseTagMixin, IApiReleaseTagMixinOptions } from '../mixins/ApiReleaseTagMixin';
+import { ApiResultTypeMixin, IApiResultTypeMixinOptions } from '../mixins/ApiResultTypeMixin';
+import { ApiPropertyItem, IApiPropertyItemOptions } from './ApiPropertyItem';
+
+/** @public */
+export interface IApiPropertySignatureOptions extends
+  IApiDeclarationMixinOptions,
+  IApiReleaseTagMixinOptions,
+  IApiResultTypeMixinOptions,
+  IApiPropertyItemOptions {
+}
+
+/** @public */
+export class ApiPropertySignature extends ApiDeclarationMixin(ApiReleaseTagMixin(
```

Comment:
>ApiPropertySignature [](start = 13, length = 20)

What's the difference between this an `ApiProperty`?

Reply:
Improve docs

---
In reply to: [236470873](https://github.com/Microsoft/web-build-tools/pull/952#discussion_r236470873) [](ancestors = 236470873)